gql/transport/requests.py

```diff
@@ -8,21 +8,33 @@
 
 
 class RequestsHTTPTransport(HTTPTransport):
-    def __init__(self, url, auth=None, **kwargs):
+    def __init__(self, url, auth=None, use_json=False, timeout=None, **kwargs):
+        """
+        :param url: The GraphQL URL
+        :param auth: Auth tuple or callable to enable Basic/Digest/Custom HTTP Auth
+        :param use_json: Send request body as JSON instead of form-urlencoded
+        :param timeout: Specifies a default timeout for requests (Default: None)
+        """
         super(RequestsHTTPTransport, self).__init__(url, **kwargs)
         self.auth = auth
+        self.default_timeout = timeout
+        self.use_json = use_json
 
-    def execute(self, document, variable_values=None):
+    def execute(self, document, variable_values=None, timeout=None):
```

Comment:
I think is useful to maintain the `execute` method to be uniform across all transports/clients.
So the way of executing will be always the same regarding the transport used.

For that reason I think is better to move out the timeout from here and just use `self.default_timeout`.
Thoughts?